controllers/article_controller.js

```diff
@@ -10,3 +13,9 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getAllArticles = (req, res, next) => {
+  selectAllArticles().then((article) => {
+    res.status(200).send(article);
```

Comment:
this should be sent on an object - when we receive it, if you just send 'article', then we just get an unlabelled article with a load of keys. 

Also, this controller is for getAllArticles, which suggests that it's actually an array, therefore has multiple objects in it, so labelling it 'articles' would be more appropriate.